### DISCUSSION TITLE
Astroid One : problem with Login Form

### DISCUSSION THREAD

**Discussion by No1Kl2** [templaza/astroid-framework#721] [General]
464 / 5.000
Hello

I am generally very happy with Astroid.

I have a problem with the login form site module on one PC.

I have placed the login module in the right sidebar.

When I want to log in, a page appears with the "Go home" button.

The login works perfectly under the "Cassiopeia" style.

I therefore suspect that the problem lies within the Astroid One style. But unfortunately I cannot see what the problem is...

Best regards

Norbert

**Comment by sonvnn** [comment]
@No1Kl2 Please help me enable Joomla debug and give me the screenshot of issue. 

Thank you.
